spec/features/request_to_book_spec.rb

```diff
@@ -1,6 +1,12 @@
 feature 'Request space' do
+
+  def get_date_range(start_date, end_date )
```

Comment:
this is defined in the server.rb file (part of the program) so should not be needed